NERODesign/content/OffScreen.qml

```diff
@@ -0,0 +1,303 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+Item {
+    width: 1024
+    height: 768
+    visible: true
```

Comment:
you also are going to have to link a selectedItem or something so that you can highlight the selected item.